src/uwtools/api/chgres_cube.py

```diff
@@ -1,13 +1,13 @@
 """
-API access to the uwtools chgres_cube driver.
+API access to the uwtools ``chgres_cube`` driver.
```

Comment:
Here and below, added some formatting in API modules for nicer RST -> HTML rendering.